playwright/fixtures/next.ts

```diff
@@ -126,10 +126,8 @@ const test = base.extend<NextTestFixtures, NextWorkerFixtures>({
             removeMock,
         });
 
-        // Close server when worker stops
-        await new Promise(() => {
-            stopMoxy();
-        });
```

Comment:
I can't believe this ever worked, and I'm sorry for not detecting it in code review.